packages/generator-honeycomb/app/templates/_Dockerfile

```diff
@@ -1,28 +1,30 @@
-FROM node:latest
+FROM node:alpine
 
 # Install Yarn
-RUN curl -o- -L https://yarnpkg.com/install.sh | bash
+RUN apk add --no-cache --virtual .build-deps tar curl bash gnupg \
+  && curl -o- -L https://yarnpkg.com/install.sh | bash \
+  && apk del .build-deps
 ENV PATH /root/.yarn/bin:$PATH
 
 # Create app directory
 RUN mkdir -p /code/<%= dir %>
 WORKDIR /code/<%= dir %>
 
-# Set environment variable
-ENV NODE_ENV production
-
 # Install app dependencies
 COPY package.json /code/<%= dir %>/
-RUN yarn --pure-lockfile
+RUN apk add --no-cache --virtual .app-deps python make g++ \
+  && yarn --pure-lockfile --production \
+  && yarn cache clean \
+  && apk del .app-deps
 
 # Bundle app source
 COPY . /code/<%= dir %>
 
-# Build
-RUN yarn run build
+# Set environment variable
```

Comment:
if we set the to NODE_ENV to prodution before yarn, then we won´t need --production or does yarn don´t check for ENV?